### ISSUE TITLE
TypeError: can only concatenate str (not "list") to str

### ISSUE THREAD

**Issue by mzaran-altius** [fiberseq/FIRE#12]
Noticed this issue in 0.0.2 & 0.0.3. We are using modules to load FIRE on our HPC cluster.

config.yaml is using local paths

`fire --configfile config.yaml -c4 --verbose`

The error
```
Using profile /net/module/sw/FIRE/0.0.3/FIRE-0.0.3/workflow/profiles/default and workflow specific profile /net/module/sw/FIRE/0.0.3/FIRE-0.0.3/workflow/profiles/default for setting default command line arguments.
Full Traceback (most recent call last):
  File "/net/module/sw/FIRE/0.0.2/lib/python3.11/site-packages/snakemake/__init__.py", line 671, in snakemake
    workflow.include(
  File "/net/module/sw/FIRE/0.0.2/lib/python3.11/site-packages/snakemake/workflow.py", line 1389, in include
    exec(compile(code, snakefile.get_path_or_uri(), "exec"), self.globals)
  File "/net/module/sw/FIRE/0.0.3/FIRE-0.0.3/workflow/Snakefile", line 25, in <module>
    excludes += [workflow.source_path(file) for file in files]
TypeError: can only concatenate str (not "list") to str

TypeError in file /net/module/sw/FIRE/0.0.3/FIRE-0.0.3/workflow/Snakefile, line 25:
can only concatenate str (not "list") to str
  File "/net/module/sw/FIRE/0.0.3/FIRE-0.0.3/workflow/Snakefile", line 25, in <module>
```
  
Our workaround to get past this error.

```
if not isinstance(excludes, list):
    excludes = [excludes]
excludes += [workflow.source_path(file) for file in files]
```

**Comment by mrvollger** [comment]
Can you share your config? We have not been seeing this.

**Comment by mzaran-altius** [comment]
Sure, we are using the config.yaml example found here - https://s3-us-west-2.amazonaws.com/stergachis-public1/index.html?prefix=Projects/Phased-GM12878/fire-test/

```
xxxxx/removed]$ cat config.yaml
ref: test/chr20.fa
ref_name: hg38
n_chunks: 20 # split bam file across x chunks
max_t: 4 # use X threaeds per chunk
manifest: test/config.tbl # table with samples to process
model: test/model.dat

# Force a read coverage instead of calulating it genome wide from the bam file.
# This can be useful if only a subset of the genome has reads.
# force_coverage: 30
# keep_chromosomes: chr20

#
# only used in model training
#
# regions of suspected regulatory elements, can be anything if not traing a model
dhs: None
# regions to not use when identifying null regions that should not have RE
excludes: None
  #- config/annotations/hg38.blacklist.ENCFF356LFX.bed.gz
  #- config/annotations/hg38.gap.bed
  #- config/annotations/SDs.merged.hg38.bed
```

We are using a test.sh script which is loaded with our module for users to run against the test data.
```
mkdir -p $HOME/FIRE/test
wget "https://s3-us-west-2.amazonaws.com/stergachis-public1/Projects/Phased-GM12878/fire-test/config.yaml" -P $HOME/FIRE/
wget "https://s3-us-west-2.amazonaws.com/stergachis-public1/Projects/Phased-GM12878/fire-test/chr20.fa" -P $HOME/FIRE/test/
wget "https://s3-us-west-2.amazonaws.com/stergachis-public1/Projects/Phased-GM12878/fire-test/chr20.fa.fai" -P $HOME/FIRE/test/
wget "https://s3-us-west-2.amazonaws.com/stergachis-public1/Projects/Phased-GM12878/fire-test/chr20.hifi.bam" -P $HOME/FIRE/test/
wget "https://s3-us-west-2.amazonaws.com/stergachis-public1/Projects/Phased-GM12878/fire-test/bam.bai" -P $HOME/FIRE/test/
wget "https://s3-us-west-2.amazonaws.com/stergachis-public1/Projects/Phased-GM12878/fire-test/config.tbl" -P $HOME/FIRE/test/
wget "https://s3-us-west-2.amazonaws.com/stergachis-public1/Projects/Phased-GM12878/fire-test/model.dat" -P $HOME/FIRE/test/
wget "https://s3-us-west-2.amazonaws.com/stergachis-public1/Projects/Phased-GM12878/fire-test/test.sh" -P $HOME/FIRE/test/
```

**Comment by apaul7** [comment]
I saw the same issue on my local cluster. 
I was able to get snakemake to run if you remove or comment out the `excludes: None` line in the example config file.

**Comment by mrvollger** [comment]
The test case config file has been fixed in #14 v0.0.4 so this shouldn't be an issue.
